### PR TITLE
Bump EOCV to Latest Version for SDK 6.1

### PR DESCRIPTION
**Pull request by serivesmejia**
# Pull Requests

Please note that we accept pull requests from anyone, but that does not mean it will be merged.

## What kind of change does this PR introduce?
* Refactor

## Did this PR introduce a breaking change?
* Nah (again)

__Please make sure your PR satisfies the requirements of [the contributing page](CONTRIBUTING.md)__

### CODE REVIEW

**Review by puravdatta-sudo — APPROVED** [review]
look good.

**Comment by JIceberg** [comment]
Hold on; is this the latest version for SDK 6.1? Or is there another version?

**Comment by litehed** [comment]
> Hold on; is this the latest version for SDK 6.1? Or is there another version?

Version 6.2 came out today just a few hours ago.

**Comment by serivesmejia** [comment]
> > Hold on; is this the latest version for SDK 6.1? Or is there another version?
> 
> Version 6.2 came out today just a few hours ago.

Yeah but, now that I remember it, you need 6.2 for using EOCV 1.4.3

**Comment by JIceberg** [comment]
> > Hold on; is this the latest version for SDK 6.1? Or is there another version?
> 
> Version 6.2 came out today just a few hours ago.

I mean another version of EOCV for the 6.1 SDK.

**Comment by serivesmejia** [comment]
It might be a better idea to merge this until updating to 6.2 and moving to maven central

**Comment by JIceberg** [comment]
> > > Hold on; is this the latest version for SDK 6.1? Or is there another version?
> > 
> > 
> > Version 6.2 came out today just a few hours ago.
> 
> Yeah but, now that I remember it, you need 6.2 for using EOCV 1.4.3

1.4.3 came out before 6.2 of the SDK did it not?

**Comment by serivesmejia** [comment]
> > > > Hold on; is this the latest version for SDK 6.1? Or is there another version?
> > > 
> > > 
> > > Version 6.2 came out today just a few hours ago.
> > 
> > 
> > Yeah but, now that I remember it, you need 6.2 for using EOCV 1.4.3
> 
> 1.4.3 came out before 6.2 of the SDK did it not?

Oh wait yeah 🤦‍♂️, a bit confusing to keep track of releases, well then this should be good to merge

**Comment by JIceberg** [comment]
I think 1.4.4 is the more recent one for SDK 6.1

**Comment by JIceberg** [comment]
Though, I'd imagine NPE is bundling 1.4.5 for 6.2 support as we speak :p

**Comment by serivesmejia** [comment]
> Though, I'd imagine NPE is bundling 1.4.5 for 6.2 support as we speak :p

Idt 6.2 added new features for cv-related stuff as 6.1 did with the fine-grained camera control thing but ¯\_(ツ)_/¯

File: core/vision/build.gradle [hunk 1]
```diff
@@ -82,4 +82,4 @@ artifacts {
 apply from: '../dependencies.gradle'
 repositories {
     mavenCentral()
-}
\ No newline at end of file
+}
```
Comment (JIceberg) [inline core/vision/build.gradle:85]: What was this change for? Did you run some sort of auto-format on the gradle file?

Comment (serivesmejia) [reply core/vision/build.gradle:85]: ¯\_(ツ)_/¯... I just used GitHub to make the edit

**Review by JarnaChao09 — APPROVED** [review]

**Review by JIceberg — APPROVED** [review]

**Pull request merged**